[2024-06-18T11:27:41.590+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: kafka_spark_dag.kafka_data_stream scheduled__2024-06-18T11:22:34.104021+00:00 [queued]>
[2024-06-18T11:27:41.593+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: kafka_spark_dag.kafka_data_stream scheduled__2024-06-18T11:22:34.104021+00:00 [queued]>
[2024-06-18T11:27:41.594+0000] {taskinstance.py:1361} INFO - Starting attempt 2 of 2
[2024-06-18T11:27:41.599+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): kafka_data_stream> on 2024-06-18 11:22:34.104021+00:00
[2024-06-18T11:27:41.601+0000] {standard_task_runner.py:57} INFO - Started process 2700 to run task
[2024-06-18T11:27:41.603+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'kafka_spark_dag', 'kafka_data_stream', 'scheduled__2024-06-18T11:22:34.104021+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/dag_kafka_spark.py', '--cfg-path', '/tmp/tmp9p3coklr']
[2024-06-18T11:27:41.604+0000] {standard_task_runner.py:85} INFO - Job 16: Subtask kafka_data_stream
[2024-06-18T11:27:41.625+0000] {task_command.py:416} INFO - Running <TaskInstance: kafka_spark_dag.kafka_data_stream scheduled__2024-06-18T11:22:34.104021+00:00 [running]> on host 2ad9230613d5
[2024-06-18T11:27:41.656+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='kafka_spark_dag' AIRFLOW_CTX_TASK_ID='kafka_data_stream' AIRFLOW_CTX_EXECUTION_DATE='2024-06-18T11:22:34.104021+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-18T11:22:34.104021+00:00'
[2024-06-18T11:27:41.658+0000] {conn.py:380} INFO - <BrokerConnection node_id=bootstrap-0 host=kafka:9092 <connecting> [IPv4 ('172.18.0.3', 9092)]>: connecting to kafka:9092 [('172.18.0.3', 9092) IPv4]
[2024-06-18T11:27:41.659+0000] {conn.py:1205} INFO - Probing node bootstrap-0 broker version
[2024-06-18T11:27:41.659+0000] {conn.py:410} INFO - <BrokerConnection node_id=bootstrap-0 host=kafka:9092 <connecting> [IPv4 ('172.18.0.3', 9092)]>: Connection complete.
[2024-06-18T11:27:41.763+0000] {conn.py:1267} INFO - Broker version identified as 2.5.0
[2024-06-18T11:27:41.764+0000] {conn.py:1268} INFO - Set configuration api_version=(2, 5, 0) to skip auto check_version requests on startup
[2024-06-18T11:27:41.766+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/src/kafka_client/kafka_stream_data.py", line 146, in stream
    data = get_data()
  File "/opt/airflow/dags/src/kafka_client/kafka_stream_data.py", line 111, in get_data
    config = dict_from_yaml(API)
  File "/opt/airflow/dags/src/kafka_client/kafka_stream_data.py", line 26, in dict_from_yaml
    with open(yaml_file, "r") as file:
FileNotFoundError: [Errno 2] No such file or directory: './config.yaml'
[2024-06-18T11:27:41.775+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=kafka_spark_dag, task_id=kafka_data_stream, execution_date=20240618T112234, start_date=20240618T112741, end_date=20240618T112741
[2024-06-18T11:27:41.782+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 16 for task kafka_data_stream ([Errno 2] No such file or directory: './config.yaml'; 2700)
[2024-06-18T11:27:41.825+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2024-06-18T11:27:41.850+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
